nextflow_schema.json

```diff
@@ -472,8 +483,8 @@
                     "description": "what class of samples do you want to keep (anything not matching this in the Class column will be removed)"
                 },
                 "rename_output_neg_camera": {
-                    "type": "string",
-                    "default": "true",
+                    "type": "boolean",
+                    "default": true,
                     "description": "you want to rename the files"
                 },
                 "rename_col_output_neg_camera": {
```

Comment:
some defaults are missing eg for "only_report_with_id_output_neg_camera" or "combine_replicate_column_output_neg_camera"